great_expectations/core/expectation_validation_result.py

```diff
@@ -503,13 +503,15 @@ class ExpectationSuiteValidationResult(SerializableDictDot):
         evaluation_parameters: Dict of Evaluation Parameters used to produce these results, or None.
         statistics: Dict of values describing the results.
         meta: Instance of ExpectationSuiteValidationResult, a Dict of meta values, or None.
-
+        batch_id: A unique identifier for the batch of data that was validated.
+        result_url: A URL where the results are stored.
     """  # noqa: E501
 
     def __init__(  # noqa: PLR0913
         self,
-        success: Optional[bool] = None,
-        results: Optional[list] = None,
+        success: bool,
+        results: list[ExpectationValidationResult],
+        suite_name: str,
```

Comment:
These should be guaranteed fields - what is a result if we don't have results, an origin suite, and a success metric?